modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryRelatedObjectsResourceImpl.java

```diff
@@ -135,7 +135,9 @@ public Page<Object> getCurrentObjectEntriesObjectRelationshipNamePage(
 			transform(
 				page.getItems(),
 				objectEntry -> _getRelatedObjectEntry(
-					relatedObjectDefinition, objectEntry)));
+					relatedObjectDefinition, objectEntry)),
+			Pagination.of((int)page.getPage(), (int)page.getPageSize()),
```

Comment:
This is weird as we already have the `pagination` attribute and it should be the same value :thinking:

Reply:
True.